### PR TITLE
Updated gpg output on - How to verify Tor Browser signature

### PR DESCRIPTION
**Pull request by championquizzer**
Updated gpg output on https://support.torproject.org/tbb/how-to-verify-signature/
Addressing this issue: https://gitlab.torproject.org/tpo/web/support/-/issues/123

### CODE REVIEW

File: content/tbb/how-to-verify-signature/contents.lr [hunk 1]
```diff
@@ -51,13 +51,13 @@ Import the Tor Browser Developers signing key (0xEF6E286DDA85EA2A4BA7DE684E2C6E8
 
 This should show you something like:
 
-    gpg: key 4E2C6E8793298290: public key "Tor Browser Developers (signing key) <torbrowser@torproject.org>" imported
+    gpg: key 4E2C6E8793298290: "Tor Browser Developers (signing key) <torbrowser@torproject.org>" 2 new signatures
```
Comment (gusgustavo) [inline content/tbb/how-to-verify-signature/contents.lr]: Ops, could you remove **2 new signatures** and replace with **imported**

File: content/tbb/how-to-verify-signature/contents.lr [hunk 2]
```diff
@@ -51,13 +51,13 @@ Import the Tor Browser Developers signing key (0xEF6E286DDA85EA2A4BA7DE684E2C6E8
...
     gpg: Total number processed: 1
-    gpg:               imported: 1
-    pub   rsa4096 2014-12-15 [C] [expires: 2020-08-24]
+    gpg:         new signatures: 2
```
Comment (gusgustavo) [inline content/tbb/how-to-verify-signature/contents.lr]: Same here, you can leave as the original - **imported: 1** 

I added the output of my console, and normally users will see other output.

**Review by gusgustavo — CHANGES_REQUESTED** [review]

File: content/tbb/how-to-verify-signature/contents.lr (hunk 1 above)
Comment (championquizzer) [reply content/tbb/how-to-verify-signature/contents.lr]: I have made the necessary changes in the latest commit [#557cb38](https://github.com/torproject/support/pull/162/commits/557cb380f5e319094e4c5b198334fa5f70e66e9c)

File: content/tbb/how-to-verify-signature/contents.lr [hunk 3]
```diff
@@ -51,13 +51,13 @@ Import the Tor Browser Developers signing key (0xEF6E286DDA85EA2A4BA7DE684E2C6E8
 
 This should show you something like:
 
-    gpg: key 4E2C6E8793298290: public key "Tor Browser Developers (signing key) <torbrowser@torproject.org>" imported
+    gpg: key 4E2C6E8793298290: "Tor Browser Developers (signing key) <torbrowser@torproject.org>" imported
```
Comment (gusgustavo) [inline content/tbb/how-to-verify-signature/contents.lr]: Please don't change this line ("public key" was removed).

**Review by gusgustavo — CHANGES_REQUESTED** [review]
Hi, there is one more little change to revert. Could you also squash your commits? Thanks!

**Comment by championquizzer** [comment]
Hi @gusgustavo, I made the necessary change and squashed the commits. Thanks!

**Pull request merged**